### ISSUE TITLE
Is there way to get withdrawal fees by REST API ?

### ISSUE THREAD

**Issue by volkovartem77** [bitmartexchange/bitmart-official-api-docs#4]
Fees: https://www.bitmart.com/fee/

**Comment by devsi** [comment]
I agree with this. Trading fee of 0.05% is fine to hardcode, but every coin has a potentially shifting withdrawal fee. Developing with the API currently means we have to constantly update the code every time a fee changes. Maybe it could be added to the JSON response for `symbols_details`?, or a separate endpoint for coin fees?

Almost every other exchange has fees available in the API. 

Thanks :)

**Comment by devsi** [comment]
> Hi guys,
> 
> We are working on the trading fee page. https://www.bitmart.com/fee/ will have more content in a day or two. Please stay tuned!
> 
> Thanks

Thanks, does this mean you'll have an API endpoint for it though?

**Comment by kemo993** [comment]
Hi,

any update regarding this?

Thank you.
